### ISSUE TITLE
DOM.eventGetCurrentEvent().getButton() always returns 1 during onMouseMove in Firefox.  

### ISSUE THREAD

**Issue by dankurka** [gwtproject/gwt#3191]
Originally reported on Google Code with ID 3185

```
Found in GWT Release:
1.5.3

Detailed description:

DOM.eventGetCurrentEvent().getButton() always returns 1 onMouseMove in
Firefox.  

While in IE DOM.eventGetCurrentEvent().getButton() returns correct mouse
button code.

```

Reported by `yilativs` on 2008-12-10 12:55:25

**Comment by zbynek** [comment]
Seems that this was fixed in 1.6  by using evt.button instead of evt.which https://github.com/gwtproject/gwt/commit/91987ae8afda4a10dd40cbb9714b95c0b18a86ee)

**Comment by niloc132** [comment]
Thanks - its too bad the ReviewPending status didn't translate always into ReviewPending labels.
